packages/x-components/src/components/base-dropdown.vue

```diff
@@ -31,10 +32,11 @@
         @keydown.end="highlightLastItem"
         @keydown.esc="close"
         @keydown.home="highlightFirstItem"
-        :aria-expanded="isOpen.toString()"
         class="x-dropdown__items-list"
-        role="listbox"
+        role="menu"
         tabIndex="0"
+        :aria-activedescendant="value"
```

Comment:
[Shouldn't the `aria-activedescendant` receive  an `id`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-activedescendant#values)? 
If that's the case we should build a way to auto-generate global unique ids for each one of the elements.

Reply:
changed to `:aria-activedescendant="`dropdown-option-${value}`"`